code/__HELPERS/unsorted.dm

```diff
@@ -1214,3 +1214,24 @@ proc/shift_proximity(atom/checker, atom/A, range, atom/B, newrange)
 		var/turf/F = E
 		F.proximity_checkers |= checker
 	return 1
+
+/proc/flick_overlay_better(image/I, atom/A, duration) //todo :^) change flick_overlay() to this
+	set waitfor = 0
+	if(!A || !I)
+		return
+	A.overlays |= I
+	sleep(duration)
```

Comment:
It's just a temp name because I didn't know what to call it and if it would replace the current one or not.